Firestore/Example/Tests/Integration/API/FIRCompositeIndexQueryTests.mm

```diff
@@ -122,9 +122,12 @@ - (NSString *)toHashedId:(NSString *)docId {
 // Asserts that the result of running the query while online (against the backend/emulator) is
 // the same as running it while offline. The expected document Ids are hashed to match the
 // actual document IDs created by the test helper.
-- (void)assertOnlineAndOfflineResultsMatch:(FIRQuery *)query
+- (void)assertOnlineAndOfflineResultsMatch:(FIRCollectionReference *)collection
```

Comment:
Giving this a second thought, I'm not sure if this is the correct way to do for composite indexing.  Composite indexing uses same collection for all tests, and use testID to differentiate between test cases + fix parallel running issue. it has integrated TTL to keep the collection size in control, but fetching the whole collection is still quite heavy. 

instead of collection, it should fetch a query where test id equal to a specific value.  Could you please try running the tests against prod, not emulator to see if the current composite index tests can pass with the current update?

Reply:
yeah, it's not a good idea to do a full collection get for composite index tests. We should figure out an alternative approach, and also fix this for Android (which has already [landed](https://github.com/firebase/firebase-android-sdk/pull/6841))

Reply:
Done.